src/api.ts

```diff
@@ -140,6 +140,9 @@ export interface ExpectedMediaItem {
 
 	/** Time to wait before removing file */
 	lingerTime?: Duration
+
+	/** Frame that media manager should grab for thumbnail preview */
+	previewFrame?: number
```

Comment:
How about naming it `thumbnailFrame`? Because generally the video preview for scrubbing is called _preview_ in MM

Reply:
Good idea!